datadog_checks_dev/datadog_checks/dev/tooling/__main__.py

```diff
@@ -0,0 +1,3 @@
+import sys
```

Comment:
license here is missing - let's add it everywhere. Since we are talking about cool task and tooling. It would be great to have a task that would update all files using one LICENSE.md file at the root directory of the project.

Reply:
Added!